docs/content/recipes/cors.md

```diff
@@ -34,9 +34,18 @@ func main() {
 		Debug:            true,
 	}).Handler)
 
+	upgrader := websocket.Upgrader{
+		CheckOrigin: func(r *http.Request) bool {
+			// Check against your desired domains here
+			return true
```

Comment:
probably should read something like:
```go
   return r.Host == "example.org"
```

defaulting to true is a huge security hole, as it allows any domain to open websockets to your graph.